docs/devcontainers/containers.md

```diff
@@ -366,6 +394,10 @@ If there are extensions that you would like always installed in any container, y
 ]
 ```
 
+### Avoiding extension reinstalls
+
+By default, VS Code will install extensions and VS Code Server inside the container's filesystem. While this has performance benefits over a locally mounted filesystem, the disadvantage is that VS Code will have to reinstall them on a container rebuild. If you find yourself rebuilding frequently, you can enable `dev.containers.cacheVolume` in your VS Code settings. This setting controls whether a Docker volume should be used to cache the VS Code Server and extensions.
```

Comment:
Thanks for pointing this out. Since it's enabled by default, I don't think we need this section, so I've removed it. This will also help with a bit more brevity for this doc.